static/app/utils/theme/theme.chonk.tsx

```diff
@@ -153,31 +153,30 @@ const lightColors = {
     black: '#181225',
     white: '#FFFFFF',
 
-    blurple100: '#463299',
-    blurple200: '#5E42CC',
-    blurple300: '#694BE5',
     blurple400: '#7553FF',
```

Comment:
These colors were either updated or wrong, but I've manually audited them and compared them to figma values